pkgs/top-level/all-packages.nix

```diff
@@ -15013,6 +15013,10 @@ with pkgs;
     backend = "wayland";
   };
 
+  rofi-network-manager-wayland = callPackage ../by-name/ro/rofi-network-manager/package.nix {
+    backend = "wayland";
+  };
```

Comment:
I am not a fan of those variants. If the other packages also only change the rofi in PATH we should remove them.